### PR TITLE
enhancing README with information about the relevant changes and how this information was obtained

### PR DESCRIPTION
**Pull request by Esser50K**
This change is [Reviewable](https://reviewable.io/reviews/tiny-pilot/tinypilot/896)

### CODE REVIEW

**Review by mtlynch — CHANGES_REQUESTED** [review]
Thanks for writing up these notes!

It's a bit hard to follow them because there's a bit of ambiguity in who's doing what. There are a few different actors at play here:

* the TinyPilot JS client
* Janus
* the uStreamer Janus plugin

In a lot of the text, it's ambiguous which thing we're talking about. For example:

>The `watch` command **is used to ask** the plugin to perform the initial WebRTC connection flow.

Does the TinyPilot JS client ask the plugin or does Janus?

Would it be possible to do a pass over the notes and make it explicit who's performing what action. I'd like to get rid of pronouns like "**it** sends the message" because those open the door to ambiguity. Instead, when we're talking about a message being sent, could we be explicit that it's "X sends message A to Y?"

> *[Reviewable](https://reviewable.io/reviews/tiny-pilot/tinypilot/896#-:-MuCLrY3Aq1IkuiyPIwg:b-ffbaci)* status: 0 of 1 LGTMs obtained

___
*[README.md, line 18 at r1](https://reviewable.io/reviews/tiny-pilot/tinypilot/896#-MuCKwgF2IEQtue3siEP:-MuCKwgF2IEQtue3siEQ:b72ixqq) ([raw file](https://github.com/tiny-pilot/tinypilot/blob/a8b1a910489615d419abae605fcae00b8650264d/README.md#L18)):*
> ```Markdown
> 
> To fetch the stream one must "talk" to the uStreamer plugin. Apparently it only receives 3 messages: `watch`, `start` and `stop`.
> This can be checked here: https://github.com/pikvm/ustreamer/blob/master/janus/src/plugin.c#L475
> ```

Can we change this URL to:

https://github.com/pikvm/ustreamer/blob/v4.11/janus/src/plugin.c#L475

Otherwise the line number will shift as there are changes to the `master` branch of uStreamer.

(ditto for line 27)
___
*[README.md, line 19 at r1](https://reviewable.io/reviews/tiny-pilot/tinypilot/896#-MuCKrVHALx_Z4eYXyC0:-MuCKrVI13t05ERKPdOl:b-656xfu) ([raw file](https://github.com/tiny-pilot/tinypilot/blob/a8b1a910489615d419abae605fcae00b8650264d/README.md#L19)):*
> ```Markdown
> To fetch the stream one must "talk" to the uStreamer plugin. Apparently it only receives 3 messages: `watch`, `start` and `stop`.
> This can be checked here: https://github.com/pikvm/ustreamer/blob/master/janus/src/plugin.c#L475
> This commit only shows usage of the `watch` and `start` commands.
> ```

Can we add more detail on the purpose of the `stop` command and when we'd integrate with it on the TinyPilot JS side?

**Comment by Esser50K** [comment]
I will make the requested changes.
Regarding the stop command, I would assume it is to stop the stream but I need to run some experiments to be sure of how it works so I can get the necessary information about its behaviour

**Comment by Esser50K** [comment]
I updated the document. I tried writing a technical overview first so that everybody is more aware of the communication actors. Actually, I try to not distinguish between the uStreamer plugin and Janus because Janus really just provides the communication layer around it.

I also did some experimenting with the `stop` command (but it's not code that is committable) and reported the findings there.

**Review by mtlynch — COMMENTED** [review]
:lgtm: 

Thanks!

> *[Reviewable](https://reviewable.io/reviews/tiny-pilot/tinypilot/896#-:-MuvzkTJCX6OxAhyE-u3:b-41s8zb)* status: :shipit: complete! 1 of 1 LGTMs obtained
